docs/src/main/sphinx/functions/math.md

```diff
@@ -226,12 +226,34 @@ Returns the value of `string` interpreted as a base-`radix` number.
 Returns the base-`radix` representation of `x`.
 :::
 
+## Geometrical functions
+
+:::{function} cosine_distance(array(double), array(double)) -> double
+Calculates the cosine distance of two double arrays:
+
+```sql
+SELECT cosine_distance(ARRAY[1.0, 2.0], ARRAY[3.0, 4.0]);
+-- 0.01613008990009257
+```
+:::
+
+:::{function} cosine_similarity(array(double), array(double)) -> double
+Calculates the cosine similarity of two double arrays:
+
+```sql
+SELECT cosine_similarity(ARRAY[1.0, 2.0], ARRAY[3.0, 4.0]);
+-- 0.9838699100999074
+```
+:::
```

Comment:
do we want to add that "similarity = 1 - distance" ? Feels like that just math theory so not something we need to add.

Reply:
or "s + d = 1" ..

Reply:
No need.